addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -90,9 +90,9 @@ def init(self):
                     END as is_unrated,
                     C.livechat_operator_id as partner_id
                 FROM mail_channel C
-                    JOIN mail_message M ON (M.res_id = C.id AND m.model = 'mail.channel')
+                    JOIN mail_message M ON (M.res_id = C.id AND M.model = 'mail.channel')
```

Comment:
Wow...

2 questions regarding this:

- Is it going to be applied for clients in stable? Don't quite remember when "init" is called. But if it's only at module installation/update then isn't it too late anyway?
- Is there some way to make an automated test for this?